USING.md

```diff
@@ -0,0 +1,56 @@
+
+## What is OPAM
+
+OPAM is the package manager for the OCaml programming language, the language 
+in which Coq is implemented.
+Instructions on
+[how to install OPAM](http://opam.ocaml.org/doc/Install.html)
+itself are available on the OPAM website.
+
+By following the instructions on this web page one installs the last stable
+version of Coq and all additional packages in the directory `~/opam-coq.@COQV@`.
+Instructions target an OPAM newcomer.
+
+## Using OPAM to install Coq
+
+Once the `opam` command is available, i.e. OPAM is installed, one can
+proceed as follows:
+
+    export OPAMROOT=~/opam-coq.@COQV@ OPAMJOBS=2
+    opam init -n --comp=@OCAMLV@
+    eval `opam config env`
+    opam repo add coq-released http://coq.inria.fr/opam/released
+    opam install coq.@COQV@ && opam pin add coq @COQV@
+
+One may also want to install CoqIDE.  Note that this requires GTK+ development
+files to be available on the system.
+
+    opam install coqide
+
+## Running Coq
+
+Every time a new shell is opened one has to type in the following lines
+in order to use Coq
+
+    export OPAMROOT=~/opam-coq.@COQV@ OPAMJOBS=2
+    eval `opam config env`
```

Comment:
I want the `--root` not to pollute the instructions, so it has to stay an env variable.
The point being that if you know a little about OPAM and want to use the standard root, copy pasting the instructions still works for you.

I'm trying to understand your suggestion about `-j`, my opam spits a quite ambiguous help
```
$ opam init --help

...
       -j JOBS, --jobs=JOBS
           Set the maximal number of concurrent jobs to use. You can also set
           it using the $OPAMJOBS environment variable.

       -j JOBS, --jobs=JOBS (absent=1)
           Number of jobs to use when building packages.
...
```

Are you saying that `-j` when passed to init becomes the new default for the root?